make-release.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+if [ -z "$1" ]; then
+    echo "Usage: $0 <release_version>"
+    exit 1
+fi
+
+RELEASE_VERSION=$1
+
+# Update CHANGES.md
+sed -i "s/## Unreleased/## Unreleased\n\n## $RELEASE_VERSION\n/" CHANGES.md
+echo "This description should answer the \"who needs this update?\" question."
+sleep 10
+${EDITOR:-vim} +7 CHANGES.md
+
+# Update version in Dockerfile
+sed -i "s/FROM ghcr\.io\/cnpem\/lnls-debian-epics-7:.*/FROM ghcr.io\/cnpem\/lnls-debian-epics-7:$RELEASE_VERSION AS build-image/" Dockerfile
+
+git checkout -b release-$RELEASE_VERSION
+git add CHANGES.md Dockerfile
+git commit -m "Release version ${RELEASE_VERSION}."
+
+# Create signed tag
+git tag -a -s $RELEASE_VERSION -m "$RELEASE_VERSION"
+git push origin release-$RELEASE_VERSION
+git push origin $RELEASE_VERSION
```

Comment:
We cannot do this until the PR is approved. Otherwise, the tag will trigger a CI that will build and push the images.